libsql_test.go

```diff
@@ -118,7 +119,7 @@ func (db Database) createTable() Table {
 	db.t.Cleanup(func() {
 		db.exec("DROP TABLE " + name)
 	})
-	return Table{name, db}
+	return Table{name: name, db: db}
```

Comment:
Why would this be a good idea?

Reply:
It is generally best practice, such as in old code if you change the order of the struct you will break the other codes as well but if you have keyed struct when you create them it will not break the code.
Because of that, most linters make lint errors when you don't put keys.